static/app/components/sidebar/broadcasts.tsx

```diff
@@ -28,15 +28,13 @@ type Props = CommonSidebarProps & {
 
 type State = {
   broadcasts: Broadcast[];
-  error: boolean;
```

Comment:
noticed that `error` is not being used, so I removed this too. please let me know if that is ok in this PR

Reply:
fine for me, but imho we should render a fallback in this case.